dot/parachain/collator-protocol/validator_side.go

```diff
@@ -228,6 +312,71 @@ type CollatorProtocolValidatorSide struct {
 	// Parachains we're currently assigned to. With async backing enabled
 	// this includes assignments from the implicit view.
 	currentAssignments map[parachaintypes.ParaID]uint
+
+	// state tracked per relay parent
+	perRelayParent map[common.Hash]PerRelayParent // map[replay parent]PerRelayParent
```

Comment:
remove this extra comment
```suggestion
	perRelayParent map[common.Hash]PerRelayParent
```